changelog/fragments/bump-kb.yaml

```
@@ -0,0 +1,50 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Bump kubebuilder commit in Operator-SDK. This fixes a bug where running `make test` 
+      would error because of Kubebuilder assets not being available at the right location. 
+      For more details, refer:
+      https://github.com/kubernetes-sigs/kubebuilder/pull/2272.
```

No need to go too into depth here. The migration guide has all the relevant info.
```suggestion
      Fixed a bug in `make test` caused by incorrect envtest asset setup.
```